src/enrichment_tables/mod.rs

```diff
@@ -17,11 +17,26 @@ pub mod geoip;
 #[cfg(feature = "enrichment-tables-mmdb")]
 pub mod mmdb;
 
-/// Configurable enrichment tables.
-#[configurable_component]
+/// Configuration options for an [enrichment table](https://vector.dev/docs/reference/glossary/#enrichment-tables) to be used in a
+/// [`remap`](https://vector.dev/docs/reference/configuration/transforms/remap/) transform. Currently supported are:
```

Comment:
I notice that there are some links error in the generated CUE when we want to reference pre-defined URL such as `urls.maxmind`. I decide to put the original link in Rust side since it would be renderend on both Vector docs and Rust docs.